draft-ietf-ppm-dap.md

```diff
@@ -1891,26 +1916,34 @@ The Leader proceeds as follows with each report:
    * `task_id` is the task ID
    * `agg_param` is the VDAF aggregation parameter provided by the Collector
      (see {{collect-flow}})
-   * `prev_state` is the state computed earlier by calling
-     `Vdaf.ping_pong_leader_init` or `Vdaf.ping_pong_leader_continued`
-   * `inbound` is the message in the `PrepareResp`
-
-   If `outbound != None`, then the Leader stores `state` and `outbound` and
-   proceeds to {{aggregation-leader-continuation}}. If `outbound == None`, then
-   the preparation process is complete: either `state == Rejected()`, in which
-   case the Leader rejects the report and removes it from the candidate set; or
-   `state == Finished(out_share)`, in which case preparation is complete and the
-   Leader commits `out_share` as described in {{batch-buckets}}. If commitment
-   fails, then the Leader rejects the report and removes it from the candidate
-   set.
```

Comment:
Rejection does nothing because the Helper has already committed. This will always result in a batch mismatch, unless the Leader has already ensured that the report was not replayed and the bucket was not collected.